runtime-tests/lambda/index.test.ts

```diff
@@ -967,3 +971,53 @@ describe('streamHandle function', () => {
     expect(output).toContain(jsonResponsePrelude.replace(nullSequence, ''))
   })
 })
+
+describe('getProcessor function', () => {
+  it('Should return a Processor (ALB) for an ALBProxyEvent event', () => {
+    const event: ALBProxyEvent = {
+      httpMethod: 'GET',
+      path: '/',
+      body: null,
+      isBase64Encoded: false,
+      requestContext: {
+        elb: {
+          targetGroupArn:
+            'arn:aws:elasticloadbalancing:us-east-1:123456789012:targetgroup/lambda-279XGJDqGZ5rsrHC2Fjr/49e9d65c45c6791a',
+        },
+      },
+    }
+
+    const processor = getProcessor(event)
```

Comment:
What do you think of whether we should test whether the `processor` is an instance of `ALBProcessor,` `EventV2Processor`, or `EventV1Processor`?

Reply:
I think that would be more robust assertion. Let me know if it's ok to export `EventProcessor`, `EventV1Processor`, `EventV2Processor`, and `ALBProcessor` classes from `handler.ts`. I can add `toBeInstanceOf` assertions in the tests in that case.

Reply:
> I think that would be more robust assertion.

Yes. It's okay to export these classes, and you can add the assertions. Can you proceed with it?

Reply:
Done!